src/components/access-type.js

```diff
@@ -167,7 +167,7 @@ class AccessTypeBase extends React.Component {
     let jwtResponse = await fetch(
       `/api/v1/access-token/integrations/${this.props.accessTypeBkIntegrationId}`
     );
-    const user = await this.setUser(
+    const user = jwtResponse.headers.get('x-integration-token') && await this.setUser(
       this.props.email,
```

Comment:
`this.setUser` was throwing error if `x-integration-token` is not available and the user has been set as Error Object which will be truthy in the next line.

Reply:
`this.setUser` needs to be done for logged-in as well as anonymous users. Whereas `jwtResponse.headers.get('x-integration-token')` would be present only when user is logged in. 
Hey, what was the error it was throwing?

Reply:
I think it was`accesstypeJwt must be present for logged in user` error.